Configurations.md

```diff
@@ -2200,6 +2200,14 @@ specific version of rustfmt is used in your CI, use this option.
 - **Possible values**: any published version (e.g. `"0.3.8"`)
 - **Stable**: No (tracking issue: [#3386](https://github.com/rust-lang/rustfmt/issues/3386))
 
+## `short_item_threshold`
+
+Maximum width threshold for a short item.
+
+- **Default value**: `10`
+- **Possible values**: any positive integer that is less than or equal to the value specified for [`max_width`](#max_width)
```

Comment:
I have added this in a new test in the `greater_than_max_width.rs` file. Do take a look

Reply:
> I'm wondering if it would be worthwhile to add a check after reading the rustmt config to ensure that the short_item_threshold doesn't exceed the max_width?

It's a good thought and I've had similar feelings  in the past as we've other similar option pairs (e.g. `chain_width`). However, I'd view that more as a general "fully valid config" check that we'd ideally come up with a similarly general way to handle; either way not necessary for this particular PR